src/Components/WebAssembly/Authentication.Msal/src/Models/MsalAuthenticationOptions.cs

```diff
@@ -48,5 +49,10 @@ public class MsalAuthenticationOptions
         /// Gets or sets whether or not to navigate to the login request url after a successful login.
         /// </summary>
         public bool NavigateToLoginRequestUrl { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets the set of known authority host names for the application.
+        /// </summary>
+        public IList<string> KnownAuthorities { get; set; }
```

Comment:
Can we initialize this collection to an empty list and change the logic on the JS side to check for no values specified?

Having things like this be null, trips users all the time in Startup. We want to avoid them having to initialize the list.